### DISCUSSION TITLE
AFK groups?

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#3894] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Hi, what about feature that we can set custom messages by permission for some groups, like admin group? For admins, there will be option to set custom message, like: Admin <name> is back! and for admins there will be just: <name> is back.

### Basic explanation as to why this feature is needed and/or useful to have: ###
For custom messages for another groups to change for ex. color of message etc., when it's another group. TO highly more important groups.

**Comment by CloudeLecaw** [comment]
Sounds a bit to complicated to the current afk reason System.
